inference-engine/src/mkldnn_plugin/mkldnn_graph_optimizer.cpp

```diff
@@ -1277,16 +1252,15 @@ void MKLDNNGraphOptimizer::FuseMVNAndSimpleOperation(MKLDNNGraph &graph) {
         }
     };
 
-    auto isSutableChildNode = [](MKLDNNNodePtr node) {
-        if (node->getType() == Quantize) {
-            auto* quantizeNode = dynamic_cast<MKLDNNQuantizeNode*>(node.get());
-            if (quantizeNode == nullptr)
-                THROW_IE_EXCEPTION << "CPU node with name '" << node->getName() << "' is not a Quantize node.";
-            return !quantizeNode->isBinarization();
-        } else if (node->getType() == Eltwise) {
-            return ((node->getAlgorithm() == EltwiseMulAdd) ||
-                    (node->getAlgorithm() == EltwisePrelu) ||
-                    (node->getAlgorithm() == EltwiseRelu));
+    auto isSutableChildNode = [](MKLDNNNodePtr childNode, MKLDNNNodePtr parentNode) {
+        if (childNode->getType() == FakeQuantize) {
+            auto* fakeQuantizeNode = dynamic_cast<MKLDNNFakeQuantizeNode*>(childNode.get());
+            if (fakeQuantizeNode == nullptr)
+                THROW_IE_EXCEPTION << "CPU node with name '" << childNode->getName() << "' is not a FakeQuantize node.";
+            return !fakeQuantizeNode->isBinarization();
+        } else if (childNode->getType() == Eltwise) {
+            return ((childNode->getAlgorithm() == EltwiseMulAdd || childNode->getAlgorithm() == EltwisePrelu) &&
+                    childNode->canBePerformedAsScaleShift(parentNode.get())) || childNode->getAlgorithm() == EltwiseRelu;
```

Comment:
Myabe it is better to extended for all algorithms that satisfies childNode->canBePerformedAsScaleShift(parentNode.get())?

Reply:
extended